### DISCUSSION TITLE
Hot keys for match braces (),[],{} in editor

### DISCUSSION THREAD

**Discussion by duvte** [doublecmd/doublecmd#1122] [Suggestions]
### Description

I often use the DC editor to edit the program code.
A hotkey is very much needed to search for match paired braces and brackets (), [], {}.

### Proposed technical implementation details (optional)

Add a hotkey for toggle the cursor position to match paired brace and bracket: ( ), [ ], { }.

**Comment by Skif-off** [comment]
Try [Ctrl+Shift+B](https://doublecmd.github.io/doc/en/shortcuts.html#editor).

**Comment by duvte** [comment]
Thank you for answer.
It's work.
Is it possible to add an ability (or internal command) to change the assign via [HotKeys](https://doublecmd.github.io/doc/en/configuration.html#ConfigHotKeys)?
f.e. Ctrl+] or Ctrl+[
